packages/grafana-schema/src/schema/mudball.cue

```diff
@@ -265,3 +265,13 @@ VizTooltipOptions: {
 	mode: TooltipDisplayMode
 	sort: SortOrder
 } @cuetsy(kind="interface")
+
+
+// Use UTC/GMT timezone
+TimeZoneUtc: "utc"  // TODO @cuetsy(kind="const")
```

Comment:
I added https://github.com/grafana/cuetsy/issues/73 for this, but not urgent since we have the duplicate in `@grafana/data` for now